keyboards/gmmk/pro/rev1/ansi/keymaps/vnmm/rules.mk

```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+RGB_MATRIX_ENABLE = yes
+ENCODER_ENABLE = yes
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
```

Comment:
This isn't actually needed.  The board uses STM32F303, and defaults to wear-leveling->embedded flash: 
https://github.com/qmk/qmk_firmware/blob/6d965d486c08166c6d6d8d699721a02b053d35d0/builddefs/common_features.mk#L236-L240

```suggestion
```